pages/doc.js

```diff
@@ -1,23 +1,22 @@
 /* global docsearch:readonly */
 
-import React, { useCallback, useEffect, useState } from 'react'
-import PropTypes from 'prop-types'
-import styled from 'styled-components'
-import Error from 'next/error'
+import React, { useState, useCallback, useEffect } from 'react'
+import kebabCase from 'lodash.kebabcase'
+import Error from 'next/error' // comp
 import Router from 'next/router'
 import fetch from 'isomorphic-fetch'
-import kebabCase from 'lodash.kebabcase'
-// components
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import { structure, getItemByPath } from '../src/utils/sidebar'
+
 import Page from '../src/Page'
 import { HeadInjector } from '../src/Documentation/HeadInjector'
 import Hamburger from '../src/Hamburger'
 import SearchForm from '../src/SearchForm'
 import SidebarMenu from '../src/Documentation/SidebarMenu/SidebarMenu'
 import Markdown from '../src/Documentation/Markdown/Markdown'
 import RightPanel from '../src/Documentation/RightPanel/RightPanel'
-// sidebar data and helpers
-import { getItemByPath, structure } from '../src/utils/sidebar'
-// styles
 import { media } from '../src/styles'
```

Comment:
@iAdramelk and Ivan: Here's the sample import sort order I came up with, as explained in https://github.com/iterative/dvc.org/issues/789#issuecomment-575267234

Reply:
There's an external import group, sorted by first-used. Then there's an internal import group (only has one line, internally sorted by first-used), and an internal component import group which includes the styled components import – also sorted internally by first-used. Notice that `Error` is a component but since it's an external import it's in the first group...

Reply:
Anyway, that issue is now closes so I guess this isn't very important anymore but just wanted to follow up with you guys in case there's any further comments or ideas. Thanks

Reply:
I'm fine with this sorting. But let's wait for @iAdramelk and may be it's not a good time to make cosmetic changes to JS when we are transferring stuff - it will create conflicts and complicate merging.

Reply:
OK. I only changed this file as an example. Not planning to change any other ones since it's a manual task and there's like 40 files. This shouldn't create a conflict unless the transfer branch also modifies the imports in this file (unlikely).

Reply:
Also reverted in c64d5906.